tests/functionality_tests/localization_visualizer.py

```diff
@@ -113,6 +113,11 @@ class DataType(IntEnum):
             frames = min(len(imu_readings) - 1, len(gps_readings) - 1)
 
     # EKF setup
+
+    # First GPS reading -> 0,0 in global frame 
+    # Map GPS coordinates to global 
+    # MAp GPS initial to 0,0
+    # New measurement - initial -> convert to meters 
```

Comment:
```suggestion
    # First GPS reading -> 0,0 in global frame 
    # Map GPS coordinates to global 
    # Map GPS initial to 0,0
    # New measurement - initial -> convert to meters 
```
Also is this an overview of the below code under `if use_ekf`? If so, maybe add another comment on top noting the context of these steps like "#Calculate initial mu and sigma for EKF initialization" (not sure if this is right)?

Reply:
added!